doc/sphinx/practical-tools/coqide.rst

```diff
@@ -330,3 +330,182 @@ If you choose something else than UTF-8, then missing characters will
 be written encoded by `\x{....}` or `\x{........}` where each dot is
 an hexadecimal digit: the number between braces is the hexadecimal
 Unicode index for the missing character.
+
+.. _coqide-debugger:
+
+Debugger
+--------
+
+Version 8.15 introduces a visual debugger for |Ltac| tactics within
+CoqIDE.  It supports setting breakpoints visually and automatically
+displaying the stopping point in the source code with "continue",
+"step over" "step in" and "step out" operations.  The call stack and variable
+values for each stack frame are shown in a new panel.
+
+The debugger is based on the non-visual |Ltac| :ref:`debugger <interactive-debugger>`.
+We'd like to eventually support other scripting facilities such as Ltac2.
+
+Since the visual debugger is new, you may encounter bugs or usability issues.
+The behavior and user interface will evolve as the debugger is refined.
+There are notes on bugs and potential enhancements at the end of
+`this page <https://github.com/coq/coq/wiki/Ltac-Debugger-Preview>`.
```

Comment:
Got it.  I wonder if it would be worthwhile to use regular expressions to flag malformed Sphinx items in CI (in the lint or the refman jobs).  That might save some tedium for reviews.

Reply:
Yeah, in fact maybe a Sphinx linter already exists?